lib/offsite_payments/integrations/gestpay.rb

```diff
@@ -189,6 +189,8 @@ def decrypt_data(shop_login, encrypted_string)
           response = ssl_get(Gestpay.service_url, decryption_query_string(shop_login, encrypted_string))
           encoded_response = parse_response(response)
           parse_delimited_string(encoded_response, DELIMITER, true)
+        rescue GestpayEncryptionResponseError => e
```

Comment:
Actually, it looks like we don't have `OffsitePayments` errors - I'll add it to the to-do list.